source/javascripts/_BitriseWorkflowEditor.js.erb

```diff
@@ -68,4 +59,11 @@
 			$httpProvider.defaults.xsrfHeaderName = "X-CSRF-TOKEN";
 		}
 	]);
+
+	BitriseWorkflowEditor.factory("$exceptionHandler", function($log, loggerService) {
+		return function myExceptionHandler(error) {
+			loggerService.error(error.message, error);
+			$log.error(error);
```

Comment:
what does `$log` do?

Reply:
This is the default functionality for the exception handler, it just logs to the browser console by default.

Reply:
do we want it in production?

Reply:
I'd say so, at least until we have a stable release and monitoring solution that has proven we actually see errors on our end.

Reply:
ok good point